web/tests/test_message_students.py

```
@@ -0,0 +1,168 @@
+        for email in mail.outbox:
+            self.assertIn(self.course.title, email.subject)
+            self.assertIn(message_data["title"], email.subject)
+            self.assertTrue(email.body)
+            self.assertNotEqual(email.body, message_data["message"])
+            try:
+                decrypted_email = decrypt_message(email.body.encode("utf-8"))
+                self.assertEqual(decrypted_email, message_data["message"])
+            except Exception as e:
+                self.fail(f"Failed to decrypt email content: {e}")
+
+    def test_send_message_to_specific_student(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_send_message_to_specific_student`

Add return type annotation: `None`

(ANN201)